tests/all/resin_root_part/resin_root_part

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+
+CONF=/mnt/boot/resinOS_uEnv.txt
```

Comment:
We should add support for rpi too. These get configured with a cmdline.txt which contains the root kernel argument.